src/main/java/seedu/timetable/Timetable.java

```diff
@@ -0,0 +1,18 @@
+package seedu.timetable;
+
+import java.util.ArrayList;
+import seedu.module.Module;
+
+public class Timetable {
+    ArrayList<Module> moduleList = new ArrayList<>();
+
+    public void add(Module module) {
+        moduleList.add(module);
```

Comment:
Would this add to the timetable in the matter outlined by our original User Guide?